docs/content/users/quickstart.md

```diff
@@ -33,6 +33,39 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
     5. If needed, import a database with `ddev import-db --src=/path/to/db.sql.gz`.
     6. Visit the project in a browser, and then build things.
 
+=== "Moodle"
```

Comment:
Hi @ibrah3m! Some missing indentation is currently breaking the page layout, and in addition to fixing that I’d recommend a few things:

- Fix up capitalization and punctuation for the bulleted list.
- Remove the note about PHP 8 since that’s DDEV’s default, we would expect it, and the docs already cover changing the PHP version.
- Turn your admonition markup into a `!!!tip` callout that’s part of our docs setup here, also edit for clarity and link to [drud/ddev-cron](https://github.com/drud/ddev-cron) so the reader has a hint about what to investigate further.

I’d also recommend moving this tab further down the page. I’m not sure about Moodle’s usage, but I doubt it’s as prevalent as WordPress or Drupal and I assume these tabs should be roughly sorted by audience size. (The tabs are also a bit out of hand here, but that’s not your fault!)

```suggestion
    ## Moodle

    ```bash
    ddev config --create-docroot --docroot public --webserver-type apache-fpm
    mkdir moodledata
    mkdir .ddev/php
    echo "max_input_vars = 5000;" > .ddev/php/custom.ini
    ddev start
    ddev exec composer create-project moodle/moodle public
    ddev launch
    ```

    Complete installation in a web browser:

    * Choose the language.
    * For the database driver, choose MariaDB.
    * On the database credentials page, enter `db` for host, name, user, and password.
    * Finish the installation.

    !!!tip
        Moodle relies on a periodic cron job—don’t forget to set that up! See [drud/ddev-cron](https://github.com/drud/ddev-cron).
```